src/content/docs/usage/server.mdx

```diff
@@ -11,6 +11,159 @@ The primary method of interacting with the server is through the Daytona CLI, al
 ## Configuring the Server
 
 The server configuration is located in different directories depending on your operating system.
-The configuration can be viewed using the `daytona server config` command across all operating systems.
+The server configuration can be viewed by using the following command across all operating systems:
 
-Server configuration can be manually edited by using the `daytona server configure` command.
+```shell
+  daytona server config
+```
+
+Daytona server configuration contains the following properties:
```

Comment:
A few weeks ago we had a PR that synced all the properties to the view. Please update to reflect the output from the latest release.

Reply:
I updated the server properties (https://github.com/daytonaio/docs/pull/105/commits/128ae87b1ca8f59cbb8fd65fbe7e09b6e96dfdc9) and added a start/stop/restart server processes (https://github.com/daytonaio/docs/pull/105/commits/58d2786e4bbad5e96993a97c95845f487237f666) to cover all functionality and to partially address the issue of not mentioning `daytona server stop` command (https://github.com/daytonaio/docs/issues/86) anywhere but the CLI  - let me know if you want to remove the latter.